docs/services/vision/detection.md

```diff
@@ -213,32 +213,26 @@ Add the Vision Service object to the services array in your raw JSON configurati
 {{% /tab %}}
 {{< /tabs >}}
 
-Click **Save config** and head to the **Components** tab.
+Click **Save config**.
 
-## Add a camera component and a "transform" model
+## Test your detector
 
-You cannot interact directly with the [Vision Service](/services/vision/).
-To be able to interact with the Vision Service you must:
+You can test your detector with [live camera footage](#live-camera-footage) or [existing images](#existing-images).
 
-1. Configure a physical [camera component](../../../components/camera/).
-Pass the name of this camera to Vision Service methods.
-2. (Optional) To view output from the detector overlaid on images from the physical camera, configure a [transform camera](../../../components/camera/transform/).
+### Live Camera footage
 
-    After adding the component and its attributes, click **Save config**.
+If you intend to use the detector with a camera that is part of your robot, you can test your detector from the [**Control tab**](/manage/fleet/robots/#control) or with code:
 
-    Wait for the robot to reload, and then go to the **Control** tab and open the transform camera card to test the stream of detections.
+1. Configure a physical [camera component](../../../components/camera/).
+2. If you would like to see detections from the **Control tab**, configure a [transform camera](../../../components/camera/transform/) to view output from the detector overlaid on images from the physical camera.
+   If you only want to access detections from code, you do not need to configure a transform camera.
+3. After adding the components and their attributes, click **Save config**.
+4. Navigate to the **Control** tab, click on your transform camera and toggle it on.
+   The transform camera will now show detections with bounding boxes around the object.
 
 ![Viam app control tab interface showing bounding boxes around two office chairs, both labeled "chair" with confidence score "0.50."](/services/vision/chair-detector.png)
 
-## Code
-
-The following code gets the robot’s vision service and then runs a color detector vision model on output from the robot's camera `"cam1"`.
-
-{{% alert title="Tip" color="tip" %}}
-
-Pass the name of the physical camera you configured, _not_ the name of the transform camera, to the Vision Service methods.
-
-{{% /alert %}}
+The following code gets the robot’s vision service and then runs a color detector vision model on output from the robot's camera `"cam1"`:
```

Comment:
```suggestion
5. To access detections with code, use the Vision Service methods on the physical camera you configured (not the transform camera).
The following code gets the robot’s vision service and then runs a color detector vision model on output from the robot's camera `"cam1"`:
```
I think it's a bit odd because we have three options (live stream control tab, live stream code, and preexisting images with code), but  they're sort of lumped into 2 instead. I think making this a step may help it make more sense.